### ISSUE TITLE
Add ember-datatables in ember-engine

### ISSUE THREAD

**Issue by tkeBe** [ember-engines/ember-engines#352]
In our core application, we added ember-datatables via `ember install ember-datatables`and the whole thing was directly working even in the templates. After installing ember-datatables in the same way inside an engine, none of the functionality was working. The error message `data-table is not a helper` appeared. 
After trying to integrate the service from the core into the engine we failed again 

![bildschirmfoto 2017-03-03 um 15 11 44](https://cloud.githubusercontent.com/assets/16797364/23554064/ccb62a64-0023-11e7-8b88-84c0985f466b.png)

If someone has had the same failure or has any idea how to solve this problem please refer to this issue.

**Comment by alexander-alvarez** [comment]
you have to declare the service as a dependency 
https://github.com/ember-engines/ember-engines#declaring-dependencies
http://ember-engines.com/guide/services

**Comment by trentmwillis** [comment]
@tkeBe after trying what @alexander-alvarez points out above, if it is still not working a reproduction of your issue will best help us figure out what is wrong. It is likely some issue in your setup or the other addon.

**Comment by knownasilya** [comment]
Helpers need to be reexported in the `app/` folder for them to work in engines..

**Comment by tkeBe** [comment]
Thank you all so far. I think by moving the ember-datatables from devDependencies to the dependencies (engine/package.json) the first part of the problem was solved. Sadly it is still not working because the bower_components belonging to the ember-datatables are not loaded now, although I installed them in the engine and in the core application. See below the new error message

`:frontend has failed to build: Error: ENOENT: no such file or directory, stat '/core/..../js/jquery.dataTables.js' `

@trentmwillis thank you. Sadly it is not that easy to reproduce the whole scenario but if it is still not working in a few days I will give it a try.

@knownasilya Not sure what you mean with reexporting ? Do you have a similar issue or a part in the documentation ?

**Comment by badmacstone** [comment]
I have a different problem with data-table and ember-engine.

This is how I use data-table:
```
{{data-table data=model
             columns=(array
               (hash title='Type' property='type')
               (hash title='Date property='createdAt')
             )
}}
```
In my engine I've got this error: `array is not a helper`. So I added `ember-array-helper` with yarn. After that I'm getting no errors but the table remains empty. Placed where the rows should be I see this: `<(unknown mixin):ember710>`. 

Any ideas how to fix this or how I get the name of the unknown mixin?

**Comment by PoslinskiNet** [comment]
@badmacstone you need to use a newer version of ember (it is included in ember from 3.7) or add specific addon that introduced this helper (for instance `"ember-composable-helpers": "*"` or another to you package.json in your engine).

**Comment by rwjblue** [comment]
Closing, I think we have veered a bit off topic of the original issue (which seems to have been resolved).
